cloud/aws/docker/lambda-handler.py

```diff
@@ -40,6 +40,10 @@ def handler(event, context):
     logging.debug("event: %s", event)
     src_cmd = base64.b64decode(event["cmd"]).decode("utf-8")
     logging.info("src_cmd: %s", src_cmd)
+    if "env" in event:
+        logging.info("env: %s", event["env"])
+        for (k, v) in event["env"].items():
+            os.environ[k] = v
```

Comment:
This is a trick to pass variables around. Because the command `src_cmd` is base64 encoded to make it serializable into the JSON, it is actually easier to provide a static command via an EventBridge transform and provide the dynamic part (for example the new object key) through a dynamically set env variable.